frontend/components/LabelsSection.js

```diff
@@ -0,0 +1,26 @@
+import { View, Text, StyleSheet } from 'react-native'
```

Comment:
nothing special here, just extracted this logic into separate component to clear Note component a little bit